meta-aos-vm-common/recipes-aos/aos-updatemanager/aos-updatemanager_git.bbappend

```diff
@@ -34,9 +34,10 @@ do_install_append() {
 python do_update_config() {
     import json
 
-    file_name = oe.path.join(d.getVar("D"), d.getVar("sysconfdir"), "aos", "aos_updatemanager.cfg")
+    file_name_source = oe.path.join(d.getVar("WORKDIR"), "aos_updatemanager.cfg")
```

Comment:
It is not good idea. The file on rootfs should be modified. Otherwise, you will break bbappend mechanism.
1. You should not use yocto for development (rebuild um manually); You could build SDK and use it for debugging and developmentl
2. If you still rebuild, do cleanall before.

Reply:
in case of UM update wi will have incorrect component id